packages/@uppy/robodog/bundle.js

```diff
@@ -1 +1,2 @@
-module.exports = require('./bundle-legacy.js')
+// eslint-disable-next-line no-multi-assign
+globalThis.Robodog = module.exports = require('./bundle-legacy.js')
```

Comment:
Do we have to use legacy / heavier bundle?

Reply:
Robodog bundle has never been cleansed from polyfills, that would be a breaking change.